cluster-autoscaler/utils/drain/drain.go

```diff
@@ -224,3 +233,18 @@ func HasLocalStorage(pod *apiv1.Pod) bool {
 func isLocalVolume(volume *apiv1.Volume) bool {
 	return volume.HostPath != nil || volume.EmptyDir != nil
 }
+
+func checkPDBs(pod *apiv1.Pod, pdbs []*policyv1.PodDisruptionBudget) (bool, error) {
```

Comment:
This only checks if there is a matching pdb, not if it has non-zero allowed disruptions. I assume we check it later? If so can we add a comment somewhere mentioning that?